english/gitlost/gitlost.md

```diff
@@ -0,0 +1,29 @@
+You find yourself lost in a GitHub tutorial....
+
+Look around, what do you see? Lots of windows open
+
+* Your Terminal
+* GitHub in one browser
+* GitHub in a different browser
+* GitGub Desktop
+* A YouTube browser showing other tutorials you might watch to help get unstuck
+
+You are psyched that you found [Linus Torvalds presenting Git to Googlei](https://youtu.be/4XpnKHJAok8) and insulting their
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.